src/libponyrt/platform/threads.c

```diff
@@ -181,7 +181,7 @@ bool ponyint_thread_create(pony_thread_id_t* thread, thread_fn start,
   // Let's use RLIMIT_STACK's current limit if it is sane.
   if(getrlimit(RLIMIT_STACK, &limit) == 0 &&
     limit.rlim_cur != RLIM_INFINITY &&
-    limit.rlim_cur >= PTHREAD_STACK_MIN)
+    limit.rlim_cur >= (unsigned long)PTHREAD_STACK_MIN)
```

Comment:
Appears that this breaks FreeBSD so we need define for freebsd.

/tmp/cirrus-ci-build/src/libponyrt/platform/threads.c:184:20: error: comparison of integers of different signs: 'rlim_t' (aka 'long') and 'unsigned long' [-Werror,-Wsign-compare]
    limit.rlim_cur >= (unsigned long)PTHREAD_STACK_MIN)
    ~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

Reply:
Fixed that.